app/src/main/java/fr/free/nrw/commons/category/CategoryImagesActivity.java

```diff
@@ -154,6 +157,20 @@ public static void startYourself(Context context, String title, String categoryN
         context.startActivity(intent);
     }
 
+    /**
+     * Consumers should be simply using this method to use this activity.
+     * This opens the activity with the standard explore categoryName
+     * @param context A Context of the application package implementing this class.
+     * @param title Page title
+     */
+    public static void startYourself(Context context, String title) {
+        if (BuildConfig.FLAVOR.equals("beta")) {
```

Comment:
IMO, it would be better to change the logic in `NavigationBaseActivity` itself and keep the method signature for `startYourself` intact. Potentially, this can be used at other places to launch the activity for any category(like POTD, UploadedWithAndroidApp etc).

Reply:
This is still possible, this is an overloaded version of the method (see line 152). All this method does is call CategoryImagesActivity with the appropriate fallback categories.

Personally I didn't think it made that much sense storing data which was for the 'fallback' activity of `CategoryImagesActivity` in `NavigationBaseActivity`, but if you think I should I can move it back.

Reply:
Sorry, I didn't realize that `startYourself(Context context, String title)` has been overloaded. Looks good to me. :)

Reply:
No problem - thanks for the review, good to keep code quality high :)